src/app/public/modules/file-attachment/file-drop.component.ts

```diff
@@ -130,9 +130,15 @@ export class SkyFileDropComponent {
     this.acceptedOver = false;
 
     if (dropEvent.dataTransfer && dropEvent.dataTransfer.files) {
-      if (this.fileAttachmentService.verifyDropFiles(dropEvent.dataTransfer.files)) {
-        this.handleFiles(dropEvent.dataTransfer.files);
+      if (!this.multiple && dropEvent.dataTransfer.files.length > 1) {
```

Comment:
Do you mind setting these statements' values to well-named variables so that their purposes are easy to read going forward? If you don't like that, maybe some helpful comments above each `if` check?
```
const isCheckingWhateverThisIsChecking = (!this.multiple && dropEvent.dataTransfer.files.length > 1);
const hasDirectory = this.fileAttachmentService.hasFolder(dropEvent.dataTransfer.files);

if (isCheckingWhateverThisIsChecking || hasDirectory) {
  return;
}

this.handleFiles(dropEvent.dataTransfer.files);
```

Reply:
Good idea. Done.